CONTRIBUTING.md

```diff
@@ -61,3 +62,75 @@ MatcherAssert.assertThat("Some reason", target, matcher);
 MatcherAssert.assertThat("Reasone one", target1, matcher1);
 MatcherAssert.assertThat("Reason two", target2, matcher2);
 ```
+
+## Pull request style
+
+Primary PR rule: it's the responsibility of PR author to bring the changes to the master branch.
+
+Other important mandatory rule - it should reference some ticket. The only exception is a minor type fix in documentation.
+
+Pull request (PR) should consist of two mandatory parts:
+ - "Title" - says **what** is the change, it should be one small and full enough sentence with only neccessary information
+ - "Description" - says **how** this pull requests fixes a problem or implements a new feature
+
+### Title
+
+Title should be as small as possible but provide full enough information to understand what was done (not a process), and where from this sentence.
+It could be capitalized If needed, started from capital letter, and should not include links or references (including tickets numbers).
+
```

Comment:
@g4s8 Why issue numbers in title is bad? Probably I used to writing number because it allows to see PRs to a single question from the list of them

Reply:
@g4s8 I don't remember all numbers of tickets. I didn't pay attention to moving mouse over PR, but yes it works for me. Now I will use this for a superficial view.